cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -373,7 +443,8 @@ profiles:
 					AllowCacheTTL:                10 * time.Second,
 					DenyCacheTTL:                 10 * time.Second,
 					RemoteKubeConfigFileOptional: true,
-					AlwaysAllowPaths:             []string{"/healthz"}, // note: this does not match /healthz/ or /healthz/*
+					AlwaysAllowPaths:             []string{"/healthz", "/readyz", "/livez"}, // note: this does not match /healthz/ or /healthz/*
+					AlwaysAllowGroups:            []string{"system:masters"},
```

Comment:
@hprateek43 I changed this back to make sure v1beta1 and v1beta2 tests were similar. Was there a reason for your change that I missed?